content/coronavirus_landing_page.yml

```
@@ -198,8 +198,8 @@ content:
     email_link: "Sign up to get emails when we change any coronavirus information on the GOV.UK website"
   live_stream:
     title: Live press conference
-    video_url: ""
+    video_url: https://www.youtube.com/watch?v=BwJPn8miKuI
```

does this need to be in

```suggestion
    video_url: "https://www.youtube.com/watch?v=BwJPn8miKuI"
```